styles/styles.css

```diff
@@ -105,6 +105,27 @@
   --font-weight-semibold: 600;
   --font-weight-bold: 700;
   --font-weight-bolder: 800;
+
+  /* margin and fine spaces */
+  --space-1px: 0.006rem;
+  --space-2px: 0.13rem;
+  --space-3px: 0.19rem;
+  --space-4px: 0.25rem;
+  --space-5px: 0.313rem;
+  --space-6px: 0.375rem;
+  --space-7px: 0.438rem;
+  --space-8px: 0.5rem;
+  --space-9px: 0.563rem;
+  --space-10px: 0.625rem;
+  --space-11px: 0.688rem;
+  --space-12px: 0.75rem;
+  --space-13px: 0.813rem;
+  --space-14px: 0.875rem;
+  --space-15px: 0.938rem;
+  --space-16px: 1rem;
+  --space-24px: 1.5rem;
+  --space-32px: 2rem;
```

Comment:
Nit: I think we can remove the last 3 which are easy to forget to use^ ^

Reply:
done